### DISCUSSION TITLE
download failed

### DISCUSSION THREAD

**Discussion by akmalalvig** [aclap-dev/video-downloadhelper#680] [Help and Q&A]
### Your question

it detect the file to download but download failed error shows

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "details": "SideDownload error: : [hls @ 0000002ee983b600] Error when loading first segment 'https://d2a5xnk4s7n8a6.cloudfront.net/spees/w/o/5d56554de4b0a9cdc810f7d0/v/62545b590cf268e0d563ebb9/u/611b77f30cf25e3afae8a6fd/t/9372a44cbf62219347946f5cc3a9ae43/p/assets/videos/5d56554de4b0a9cdc810f7d0/2022/04/11/62545b590cf268e0d563ebb9/hls_1M_000.ts'\r\n[in#0 @ 0000002ee983b440] Error opening input: Invalid data found when processing input\r\nError opening input file https://d2a5xnk4s7n8a6.cloudfront.net/spees/w/o/5d56554de4b0a9cdc810f7d0/v/62545b590cf268e0d563ebb9/u/611b77f30cf25e3afae8a6fd/t/9372a44cbf62219347946f5cc3a9ae43/p/assets/videos/5d56554de4b0a9cdc810f7d0/2022/04/11/62545b590cf268e0d563ebb9/hls_1M_.m3u8.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_5746485758311423",
    "error": "coapp_failure",
    "id": "coapp_failure_3dfe60e4-bb3e-4cd1-a74c-70b2a1959301"
  }
}

**Comment by Mattmjs** [comment]
Another "unable to open key file".
